content/SCALECLIReference/Storage/CLIDisk.md

```diff
@@ -11,7 +11,624 @@ tags:
+| `hddstandby`     | HDD standby timer.                             |
+| `advpowermgmt`   | Advanced power management profile.             |
+| `togglesmart`    | S.M.A.R.T. status.                             |
+| `smartoptions`   | Applied S.M.A.R.T. options.                    |
+| `expiretime`     | <!--Not sure about this one.-->                |
+| `critical`       | Threshold temperature in Celsius               |
+| `difference`     | Report if the temperature of a drive has changed by this many degrees Celsius since the last report. |
+| `informational`  | Report if drive temperature is at or above this temperature in Celsius. 0 disables the report. |
+| `model`          | Disk model number.                             |
+| `rotationrate`   | Disk rotation rate.                            |
+| `type`           | Disk type (HDD, SSD, NVME).                    |
+| `zfs_guid`       | Disk ZFS ID.                                   |
+| `bus`            | Disk bus type (ATA, SCSI, M.2).                |
+| `devname`        | Disk developer name.                           |
+| `enclosure`      | Disk enclosure.                                |
+| `supports_smart` | Disk S.M.A.R.T. support status.                |
+| `pool`           | Disk pool.                                     |
+{{< /truetable >}}
+{{< /expand >}}
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>storage disk query <i>attribute</i></code>
+
+Where:
+* *attribute* is the attribute you want to filter in the query.
+
+{{< expand "Command Example" "v" >}}
+```
+storage disk query name
++---------+
+| name    |
++---------+
+| sdb     |
+| sda     |
+| nvme0n1 |
+| sdc     |
+| sdd     |
+| sdm     |
+| sdn     |
+| sde     |
+| sdf     |
+| sdg     |
+| sdh     |
+| sdi     |
+| sdj     |
+| sdk     |
+| sdl     |
++---------+
+```
+{{< /expand >}}
+{{< /expand >}}
+
+### Resize Command
+
+The `resize` command allows you to resize disks to a specific amount of gigabytes.
+
+{{< expand "Resizing Disks">}}
+
+#### Description
+The `resize` command has three properties. They are `disks`, `sync`, and `raise_error`.
+Enter the command string, then press <kbd>Enter</kbd>.
+The command returns completion percentages when successful.
+
+#### Usage
+
+From the CLI prompt, enter:
+
+<code>storage disk resize disks={"name":"<i>diskname</i>,<i>diskname</i>", "size":"<i>number</i>"} sync=<i>true/false</i> raise_error=<i>true/false</i><code>
+
+Where:
+* *diskname* is the name of the disk you want to resize. Separate multiple disks with commas
+* *number* is the size (in gigabytes) you want to resize the disk to.
+* *true/false* in the `sync` property enables or disables synchronizing the new size of the disk(s) with the database cache. The default is true.
+* *true/false* in the `raise_error` property sets whether the disk(s) raise a CallError upon failure (true) or will only log errors (false). The default is false.
```

Comment:
do not use will, and this detailed explanation goes in in a table describing the property and how to enter the argument. The Where section is about explaining the variable not the arguments.